lib/component.js

```diff
@@ -217,7 +217,7 @@ class Component extends WebComponent {
     // initialize shared state store, either in `appState` or default to `state`
     // appState and isStateShared of child components will be overwritten by parent/root
     // when the component is connected to the hierarchy
-    this.state = {};
+    this.state = Object.assign({}, this.getConfig(`defaultState`));
```

Comment:
Can you also remove the other usage of `defaultState` in `connectedCallback`: https://github.com/mixpanel/panel/blob/06e2194a99e5b105ca3e5349f054060e990ab319/lib/component.js#L288.

Reply:
9e78237